### ISSUE TITLE
NLP based project to predict if a restaurant review is positive or negative

### ISSUE THREAD

**Issue by geekaditi** [Jayshah6699/datascience-mashup#43]
It will use the bag of words model and Naive Bayes classification. The code will be in python.

**Comment by Jayshah6699** [comment]
I am gonna need dataset information and a little bit more information about your approach too @geekaditi

**Comment by geekaditi** [comment]
I have the dataset with me in a .tsv file. It has 2 columns. One is for review and the other one is in binary-1 for positive review and 0 for negative. It has a total of 1000 reviews in the file. My approach is first cleaning the text(libraries used: re,nltk). Next I'll create the bag of words model using the countVectorizer class from the sklearn library.Then split the dataset into train and test set. And fit the Gaussian naive bayes classifier to the training set. Then make predictions using this classifier on the test set.  Can I make a PR with the code and dataset? That way it'll be easier for you to understand the approach.

**Comment by ksdkamesh99** [comment]
I want to work on this Kindly assign me if @geekaditi takes more time @Jayshah6699

**Comment by Jayshah6699** [comment]
Everyone takes their own time some people can do it fast some people take time for learning and then apply.
@ksdkamesh99 kindly do another project as there are thousands of similar projects on different datasets with a little bit of changes and a little different approach 
So I would suggest let's @geekaditi work on this project and you pick another one
